src/features/movies/MovieDetails/MovieInfo/styled.js

```diff
@@ -0,0 +1,257 @@
+import styled from "styled-components";
+import { ReactComponent as StarIcon } from "../../../../images/star.svg";
+import { ReactComponent as NoImageIcon } from "../../../../images/no-movie-poster.svg";
+
+export const MovieTile = styled.article`
+  background: ${({ theme }) => theme.colors.white};
+  padding: 40px;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+  overflow: hidden;
+  margin-bottom: 64px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}){
+    padding: 28px;
+    margin-bottom: 32px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}){
+    padding: 16px;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 21px;
+  }
+`;
+
+export const Wrapper = styled.span``;
```

Comment:
Span is not a block element, therefore it shouldn't be a parent. From the semantic point it should be, for example, div or another block element.

Reply:
Since this pull request needed to be merged already, I'll fix that in the next pull request 😀

Reply:
👍